ui/packages/consul-ui/app/components/consul/peer/list/index.hbs

```diff
@@ -0,0 +1,93 @@
+<ListCollection
+  class="consul-peer-list"
+  ...attributes
+  @items={{@items}}
+  @linkable="linkable peer"
+as |item index|>
+  <BlockSlot @name="header">
+{{#if (can 'delete peer' item=item)}}
+      <a
+        data-test-peer={{item.Name}}
+        href={{href-to 'dc.peers.edit' item.Name}}
+      >
+        {{item.Name}}
+      </a>
+{{else}}
+      <p>
+        {{item.Name}}
+      </p>
+{{/if}}
+  </BlockSlot>
+  <BlockSlot @name="details">
+    <div class="peers__list__peer-detail">
+      <Peerings::Badge @peering={{item}} />
+
+      <div
+        {{tooltip
+          (t 'routes.dc.peers.index.detail.imported.tooltip'
+            name=item.Name
+          )
+        }}
+      >
+        {{t 'routes.dc.peers.index.detail.imported.count'
+          count=(format-number item.ImportedServiceCount)
+        }}
+      </div>
+
+      <div
+        {{tooltip
+          (t 'routes.dc.peers.index.detail.exported.tooltip'
+            name=item.Name
+          )
+        }}
+      >
+        {{t 'routes.dc.peers.index.detail.exported.count'
+          count=(format-number item.ExportedServiceCount)
+        }}
+      </div>
+
+    </div>
+  </BlockSlot>
+  <BlockSlot @name="actions" as |Actions|>
+{{#if (can 'delete peer' item=item)}}
+
+    <Actions as |Action|>
+      <Action
+        data-test-edit-action
+        @href={{href-to 'dc.peers.edit' item.Name}}
+      >
+        <BlockSlot @name="label">
+          View
+        </BlockSlot>
+      </Action>
+      <Action
+        data-test-delete-action
+        @onclick={{action @ondelete item}}
```

Comment:
`{{fn @onDelete item}}` is more idiomatic in Octane

Reply:
Do you know if there is a codemod to do this? I'd like to get rid of all our `action` things now if possible, now that we have `pick` we don't need `value=""` ability that `action` has.

In the meantime will switch this one up, it was just copied from elsewhere.

Reply:
Switched up this one `action` > `fn`